### PR TITLE
DUPLO-14355 exporter for k8-job and k8-cron-job

### PR DESCRIPTION
**Pull request by Duplo-nikv**

### CODE REVIEW

File: .envrc
Comment (tahir-duplo) [inline .envrc:1]: Remove secrets from this file.

File: .vscode/launch.json
Comment (tahir-duplo) [inline .vscode/launch.json:1]: Remove this file from github.

Comment (Duplo-nikv) [reply .vscode/launch.json:1]: done

**Review by tahir-duplo — APPROVED** [review]
LGTM

**Pull request merged**